lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -91,13 +91,14 @@ llvm::Error HostManager::addNetwork(std::unique_ptr<Module> module,
       RETURN_IF_ERR(::glow::optimizeFunction(F, *backend_, cctx));
     }
   }
-  auto partitioner = Partitioner(module.get(), deviceInfo, saturateHost);
+  auto partitioner =
```

Comment:
Can you put a comment here saying that if profiling then the Partitioner should only return a single partition? Perhaps could add an assert for this too?

Reply:
I'll add a comment, perhaps return an error rather than assert?